storage/api/Storage/Storage.csproj

```diff
@@ -2,11 +2,11 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
Change this to netcoreapp2.1.  2.2 is already passed its end of life.  2.1 will be maintained through 2020.